src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx

```diff
@@ -104,19 +114,28 @@ const ConfirmTransactionModal = ({
               </div>
             </div>
             <div className="flex justify-between mb-2 font-bold">
-              <div className="text-secondary-alternate">Shuttle fee (estimated)</div>
+              <div className="text-secondary-alternate">Estimated Lower Bound Shuttle fee</div>
```

Comment:
Casing doesnt seem to be right
```suggestion
              <div className="text-secondary-alternate">Estimated Lower Bound Shuttle Fee</div>
```

or `Estimated lower bound shuttle fee`